survey/tests/testdump.json

```diff
@@ -75,6 +47,18 @@
   "template": ""
  }
 },
+{
+ "model": "survey.survey",
+ "pk": 3,
+ "fields": {
+  "name": "Unicode Survey",
+  "description": "This is a survey to test unicode choices.",
```

Comment:
Could you make a survey in chinese  instead of english ? It can be very small, 2 question with 2 answers each are more than enough.